docs/plugins/typescript-react-apollo.md

```diff
@@ -52,13 +52,13 @@ Or if you prefer:
 Customize from where will be `gql` imported. This is useful if you want to use eg. `graphql.macro` for inlining documents.
 Note that you need to write whole import line, eg. `import { gql } from 'graphql.macro'`.
 
-#### `noHOC` (default value: `false`)
+#### `withHOC` (default value: `true`)
```

Comment:
Feels like unnecessary breaking change just for switching logic around. If anything, it should still support old flags, just deprecate them and convert into these.

Reply:
I agree with @FredyC . I think it's better to change it back @ardatan :)